### A simple neural network making a prediction

In [3]:
weight = 0.05

def neural_network(input, weight):
    prediction = input * weight
    return prediction

number_of_toes = [8.5, 9.5, 10, 9]
input = number_of_toes[0]
pred = neural_network(input, weight)

print(pred)

0.42500000000000004


There is a 42.5% chance that the team will win based on the average number of toes per player on the team